.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,28 @@
+Thanks for submitting code to Blockly!
+
+## The basics
+
+- [ ] You branched from develop, not master
```

Comment:
Change these to first person, "I created my branch from develop.\This pull request is for develop.\My code follows the style guide."

Reply:
Done.